libcontainer/criu_opts.go

```diff
@@ -1,5 +1,14 @@
 package libcontainer
 
+type cg_mode uint32
+
+const (
+	CRIU_CG_MODE_SOFT cg_mode = 3 + iota
+	CRIU_CG_MODE_FULL
+	CRIU_CG_MODE_STRICT
+	CRIU_CG_MODE_DEFAULT
+)
```

Comment:
(just passing by); if that needs explanation - would it be useful to add that information as a comment in the code?